package/psplash/psplash.mk

```diff
@@ -29,6 +29,12 @@ endef
 PSPLASH_POST_EXTRACT_HOOKS += PSPLASH_COPY_IMAGE
 endif
 
+ifeq ($(BR2_PACKAGE_EMB_MACAW_CHAPS),y)
+define PSPLASH_BUILD_CMDS
+    cp -a $(@D)/base-images/psplash-chargepoint-macaw.png $(@D)/base-images/psplash-chargepoint.png
```

Comment:
patch the COPY_IMAGE above and then make that configuration happen for macaw - ie. make this part

```
ifneq ($(PSPLASH_IMAGE),)
define PSPLASH_COPY_IMAGE
	cp $(@D)/base-images/$(PSPLASH_IMAGE) $(@D)/base-images/psplash-chargepoint.png
endef
```

Then add BR2_PACKAGE_PSPLASH_IMAGE=psplash-chargepoint-800x600.png into the macaw buildroot defconfig

Reply:
I added the PSPLASH_IMAGE definition in `mcb/buildroot_defconfig` as follows:
`BR2_PACKAGE_PSPLASH_IMAGE="psplash-chargepoint-macaw.png"`
However this definition is not visible in the `psplash.mk` file.

Reply:
I added the PSPLASH_IMAGE definition in `mcb/buildroot_defconfig` as follows:
`BR2_PACKAGE_PSPLASH_IMAGE="psplash-chargepoint-macaw.png"`
However this definition is not visible in the `psplash.mk` file.